Dockerfile

```
@@ -0,0 +1,13 @@
+FROM golang:1.22.5
+
+WORKDIR /app
+
+COPY go.mod go.sum tracker.db ./
```

БД не нужно копировать, так как она не нужна для скачивания зависимостей

```suggestion
COPY go.mod go.sum ./
```